src/config/binaryClassification.ts

```diff
@@ -3,6 +3,11 @@ export const binaryClassification = ({
     description = "",
     labels = "",
     examples = [],
+}: {
+    text_input?: string;
+    description?: string;
```

Comment:
`description` is not optional. There are a few keys that are required at all times.